dracut/modules.d/90qubes-pciback/module-setup.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/bash
 
-install() {
+install () {
     inst_hook cmdline 02 "$moddir/qubes-pciback.sh"
     inst lspci
     inst grep
     inst awk
+    mkdir -p -m 0700 -- "$initdir/etc/usbguard"
+    mkdir -p -m 0755 -- "$systemdsystemunitdir/usbguard.service.d"
+    inst_multiple /etc/nsswitch.conf
+    inst_multiple /etc/usbguard/{qubes-usbguard.conf,rules.conf,rules.d,IPCAccessControl.d}
```

Comment:
`inst_multiple` appears to _not_ copy directories recursively. Resulting `/etc/usbguard/rules.d` in initramfs is empty, so of course it can't work.
Adding separate `inst_multiple /etc/usbguard/rules.d/*.conf` made it work.